src/drt/src/dr/FlexDR_init.cpp

```diff
@@ -2116,16 +2116,19 @@ void FlexDRWorker::initMazeCost_marker_route_queue_addHistoryCost(
                 }
                 case frcInstBlockage: {
                   frInst* inst = (static_cast<frInstBlockage*>(src))->getInst();
-                  std::cout << inst->getName() << "/OBS" << " ";
+                  std::cout << inst->getName() << "/OBS"
+                            << " ";
```

Comment:
weird reformatting - are you on clang-format 18.1.3?

Reply:
version 19.1.7.
Have no idea why this happens